easybuild/easyconfigs/l/libdap/libdap-3.14.0-intel-2015a-Python-2.7.10.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'libdap'
+version = '3.14.0'
+
+homepage = 'http://opendap.org/download/libdap'
+description = """A C++ SDK which contains an implementation of DAP 2.0
+ and the development versions of DAP3, up to 3.4.
+ This includes both Client- and Server-side support classes."""
+
+source_urls  = ['http://www.opendap.org/pub/source/']
+sources = [SOURCE_TAR_GZ]
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+patches = ['%(name)s-%(version)s_flex.patch']
+
+builddependencies = [
+    ('Bison', '3.0.2'),
+    ('flex', '2.5.39'),
+]
+
+pythonversion = '2.7.10'
+versionsuffix = "-Python-%s" % pythonversion
```

Comment:
`pythonversion` is used nowhere else, so just collapse it into `versionsuffix`, and move `versionsuffix` above right below `version`?